requirements.txt

```diff
@@ -1,5 +1,9 @@
+opencv-python==4.1.1.26
+pafy==0.5.5
```

Comment:
Same here, lower-bound version requirements are preferable here (if `0.5.5` is the minimum compatible version then `>=0.5.5`)

Reply:
Done!